dbfit-java/build.gradle

```diff
@@ -83,7 +83,80 @@ subprojects {
         include('**/*Support*')
     }
 
-    task integrationTest(type: Test) {
+    task deploymentProperties {
+        doLast {
+            ext {
+                jdbcDriverClass = ''
+                jdbcDriverUrlPrefix = ''
+                delimeterType = 'normal'
+                delimeter = ';'
+                setUpTestDbScript = "${rootProject.projectDir}/test_vm/scripts/${project.name}/sql/create-databases-${project.name}.sql"
+                tearDownTestDbScript = "${rootProject.projectDir}/test_vm/scripts/${project.name}/sql/drop-databases-${project.name}.sql"
+            }
+        }
+    }
+
+    // to be overriden when needed
+    ext.connectionPropertiesFile = '/dev/null'
```

Comment:
@javornikolov I'm currently reworking some of this stuff as I've realised this too since I've pushed this. Some of this stuff isn't very elegant and I'll spin off another branch to review an alternative.